Library/DiscUtils.Lvm/PhysicalVolume.cs

```diff
@@ -72,6 +72,14 @@ public static bool TryOpen(Stream content, out PhysicalVolume pv)
             return true;
         }
 
+        public static bool CanOpen(PartitionInfo volumeInfo)
+        {
+            using (var content = volumeInfo.Open())
+            {
+                return SearchLabel(content, out _);
```

Comment:
the discarded `PhysicalVolumeLabel` is only reading 512 bytes and calculating one crc cheksum